hera_pspec/pspecdata.py

```diff
@@ -2887,6 +3577,39 @@ def pspec(self, bls1, bls2, dsets, pols, n_dlys=None,
                 raise ValueError("None of the specified polarization pairs "
                                  "match that of the UVData objects")
 
+        '''
```

Comment:
This should be deleted. The code was restructured and moved up to line ~3490 but I kept the commented out portion since it was a large change and I was afraid it would break a bunch of stuff.